<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestión de Estudiantes</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>401</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_edad">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>14</number>
      </property>
      <property name="maximum">
       <number>115</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cargar">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>571</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Estudiantes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_clases">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cantidad de clases que cursó:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_gradoAcad">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Licenciatura</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Maestría</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lE_nombreEstudiante">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre del estudiante:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_notas">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Subir notas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_guardar">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Elija el grado académico que esta cursando: </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestión de Vehículos</string>
     </attribute>
     <widget class="QLineEdit" name="le_modelo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modelo:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_year">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2026</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_vehiculo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Carro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Camión</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Año</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo de Vehículo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>641</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Vehículos en una Flota</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kilometraje</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_km">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999999999999.000000000000000</double>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>431</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cargarVehi">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_vehiculo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingresar Vehículo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_guardarVehi">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Eliminar">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>450</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox_edadEmpleado">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2026</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cargarEmpleado">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo de empleado:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>641</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Empleados</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nombreEmpleado">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_salario">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999999999999.000000000000000</double>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_empleado">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Contratado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fijo</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>431</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_guardarEmpleado">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_EliminarEmpleado">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>450</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_empleado">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingresar Empleado</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Salario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_horas">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2026</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_horas">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Horas Trabajadas</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
